src/code.c

```diff
@@ -1616,6 +1654,11 @@ void CodePow ( void )
 }
 
 
+void CodeGAPSmallInt(Obj val)
```

Comment:
I'd really prefer if this was called `CodeImmediateInt` -- saying "small int" is quite ambiguous. This code instead deals with "immediate integer handles" (to use the language from `gmpints.c`)

[ Sorry, I thought I had mentioned this in my previous PR? But perhaps I failed to actually submit that comment? I can't find it now, in any case :-/ ]